task/dispatcher_test.go

```diff
@@ -1,11 +1,14 @@
 package task
 
 import (
+	"github.com/smartystreets/goconvey/convey"
```

Comment:
use . "github.com/smartystreets/goconvey/convey"
and move to outer depends block